components/scripts/lib/libs.sh

```diff
@@ -1,37 +1,52 @@
 #!/usr/bin/env bash
 
+failed_to_load_lib() {
+  local lib_name _RED _BOLD _RESTORE _UNEXPECTED_ERROR
```

Comment:
Have to prefix the local constants here with `_` so that they don't conflict with the other constants that are loaded by the color.sh and exit-code.sh library scripts.